docs/samples/pipelines/kfs-pipeline-v1alpha2.ipynb

```diff
@@ -0,0 +1,189 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# KFServing Pipeline samples\n",
+    "\n",
+    "This notebook uses an older version of the KFServing Pipelines component meant for clusters using a KFServing version less than v0.5.0 which only supports the v1alpha2 API."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install the necessary kfp library"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install kfp --upgrade"
```

Comment:
Since 0.6.0 may remove the support of v1alpha2 API, we should cap the version `kfp < 0.5.0` or  0.6.0 to avoid future errors.

Reply:
Are you thinking of the `kfserving` package in regards to the removal of v1alpha2 support? `kfp` should be fine uncapped, right?

Reply:
Yeah sorry I meant the `kfserving` package. Looks like this notebook didn't force user to run pip install kfserving.